modules/schedule/src/test/java/org/apache/ignite/internal/processors/schedule/GridScheduleSelfTest.java

```diff
@@ -243,9 +244,11 @@ public void testScheduleRunnable() throws Exception {
     public void testScheduleCallable() throws Exception {
         SchedulerFuture<Integer> fut = null;
 
-        long freq = 60; // 1 minute frequency.
+        long freq = 1; // 1 second frequency.
```

Comment:
Can you avoid changing existing test, add a new test instead, to be able to check both formats?

Reply:
I changed this just to speed up the test as with Spring we can schedule with seconds.
There is another test with a minute frequency.